client-sdk/ts-web/core/playground/src/startPlayground.mjs

```diff
@@ -7,7 +7,7 @@ export async function startPlayground() {
     {
         console.log('waiting for node to be ready');
         const waitStart = Date.now();
-        await nic.nodeControllerWaitReady();
+        await nic.nodeControllerWaitSync();
```

Comment:
Iirc you mentioned this is a workaround for due to a bug in 23.0 right? Can this be reverted with 23.1?

Reply:
Yeah, with 23.0.1, will change it :+1:

Reply:
Seems to work.